### ISSUE TITLE
Type error: Type 'Uint8Array<ArrayBufferLike>' is not assignable to type 'Uint8Array<ArrayBuffer>'.

### ISSUE THREAD

**Issue by aaalloc** [ngxson/wllama#137]
When compiling with nextjs

```
./node_modules/.pnpm/@wllama+wllama@2.0.0/node_modules/@wllama/wllama/src/wllama.ts:626:7
Type error: Type 'Uint8Array<ArrayBufferLike>' is not assignable to type 'Uint8Array<ArrayBuffer>'.
  Type 'ArrayBufferLike' is not assignable to type 'ArrayBuffer'.
    Type 'SharedArrayBuffer' is missing the following properties from type 'ArrayBuffer': resizable, resize, detached, transfer, transferToFixedLength

  624 |         break; // stop token
  625 |       }
> 626 |       outBuf = joinBuffers([outBuf, sampled.piece]);
      |       ^
  627 |       if (options.onNewToken) {
  628 |         options.onNewToken(sampled.token, sampled.piece, bufToText(outBuf), {
  629 |           abortSignal,
 ELIFECYCLE  Command failed with exit code 1.
```

tsconfig.json
```
{
  "compilerOptions": {
    "target": "ES2018",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "bundler",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ],
    "paths": {
      "@/*": ["./*"]
    }
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
  "exclude": ["node_modules"]
}
```

any idea how to fix this ?

**Comment by jvondermarck** [comment]
INSANE

**Comment by ngxson** [comment]
Should be fixed in v2.0.1
